### ISSUE TITLE
Line graph offset from category axis

### ISSUE THREAD

**Issue by igoriok1994** [amcharts/amcharts3#209]
Hi.
Maybe someone know how to offset graph from category axis?
![image](https://user-images.githubusercontent.com/37294916/47003606-5210b300-d138-11e8-8467-6b57a113edea.png)
When value = 0, line is invincible. 
O maybe there is a way to offset value axis or something like that...

![image](https://user-images.githubusercontent.com/37294916/47003771-ad42a580-d138-11e8-8dc9-f88c4a59d5d9.png)

**Comment by RobbiesCode** [comment]
Hi there,

Have you tried setting a `minimum` on your valueAxis? (You might need to enforce this setting by adding `strictMinMax: true`.

```
"valueAxes": [{
  "minimum": -0.1,
  "strictMinMax": true
}]
```

Hope this helps!

**Comment by igoriok1994** [comment]
Hi,

Thanks for suggestion I haven't tried it yet, but I will.
But what if I don't know max and min values (another case)?
Eg. I can have 100k amount of data. Is there any way to set offset from bottom (value axis, graph)?

**Comment by RobbiesCode** [comment]
That would require some extra logic. There are no settings that would do what you described, but you can set the minimum (or maximum) dynamically, based on your data, inside `addInitHandler`. This method is used before the chart is initialized.

Here is an example for a simple column chart:

```
AmCharts.addInitHandler(function(chart) {

  // find data minimum & maximum:
  var min = chart.dataProvider[0].value;
  var max = chart.dataProvider[0].value;
  for (var i in chart.dataProvider) {
    if (chart.dataProvider[i].value > max) {
      max = chart.dataProvider[i].value;
    }
    if(chart.dataProvider[i].value < min){
      min = chart.dataProvider[i].value;
    }
  }

  min -= 10; // subtract an arbitrary offset 

  // set axes max based on value above:
  chart.valueAxes[0].minimum = min;
  // chart.valueAxes[0].maximum = max;
  chart.valueAxes[0].strictMinMax = true;
});
```

**Comment by igoriok1994** [comment]
Yes, looks like I have to add min/max logic to my pivot. Thanks :)
